inception-recommendation-api/src/main/java/de/tudarmstadt/ukp/inception/recommendation/api/util/CasUtil.java

```diff
@@ -193,20 +194,23 @@ private CasUtil()
             TokenObject tObj = sentence.get(i);
 
             if (indexAnnotations >= annotations.size()) {
-                result.add(new AnnotationObject(null, tObj, sentence, id, feature));
+                result.add(new AnnotationObject(tObj, id, feature, SOURCE));
```

Comment:
Why do we need to set Source to "CasUtil" here? Wouldn't it make more sense to pass in the classifier name as a parameter... or just leaving the source empty?

Reply:
Then how about leaving the source null?

Reply:
Done